### PR TITLE
Convert to HTML error page

### PR DESCRIPTION
**Pull request by stubailo**
One big reason we need this is that the text format doesn't work on Windows
because of line endings. We could just replace the line endings based on
platform, but this seemed better overall.

### CODE REVIEW

File: tools/runners/run-proxy.js [hunk 1]
```diff
@@ -174,13 +174,37 @@ _.extend(Proxy.prototype, {
...
+    </style>
+  </head>
+
+  <body>
+    <h3>Your app is crashing. Here's the latest log:</h3>
+
+    <pre>`);
 
         _.each(runLog.getLog(), function (item) {
           c.res.write(item.message + "\n");
```
Comment (glasser) [inline tools/runners/run-proxy.js]: html-escape (&lt; and friends), I think

Comment (stubailo) [inline tools/runners/run-proxy.js]: Darn! Missed that. Do we have any utilities like this in Meteor somewhere that I could use?

Comment (glasser) [inline tools/runners/run-proxy.js]: There's one in blaze preamble.js.  Dunno about tools.

**Comment by stubailo** [comment]
Merged.

**Pull request closed without merging**